readthedocs/core/static-src/core/js/doc-embed/footer.js

```diff
@@ -22,21 +22,6 @@ function injectFooter(data) {
 }
 
 
-function setupBookmarkCSRFToken() {
```

Comment:
This isn't needed because we aren't doing POST's anymore?

Reply:
Isn't needed because we don't have the bookmark feature anymore p: